cowprotocol/accounting/rewards/mainnet/service_fee_query_4017925.sql

```diff
@@ -1,5 +1,15 @@
 with
 
+-- Add colocated solvers here
+colocated_solvers as (
+    select
+        'prod-Barter' as solver_name,
```

Comment:
note we also have barn-Barter and prod-Copium_Capital that are colocated

Reply:
Thanks, I was hoping to try filling them automatically using the CMS. If I can't set that up on time before Monday I'll just add those manually here and set this PR as ready to review.